### ISSUE TITLE
Reg\Register an Operation\Page "New Entrant Application" - External user can register the operation

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1906]
#### Description:
As an industry user, given that the operation has been added to my operator, I can register the operation for IRC approval.

#### Acceptance Criteria:
Given that I am on the Registration Information page
If I select New Entrant as the registration category
Then I get an extra breadcrumb as step number 3, after Facility Information, called "3 New Entrant Application"
https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=13653-1955&t=rBpJCzhxRzIMrjwm-4

#### Development Checklist:

- [ ] Use existing widget
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by shon-button** [comment]
@patriciarussellCAS

Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/registration/register-an-operation

**Comment by patriciarussellCAS** [comment]
Was able to review and confirm this workflow. 
Design note: Include a link to glossary as well as GGERR so they can easily find the definitions related to new entrant.
